src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
In the future there would be only one "family"... defining such renderV2() is just a transitory situation (although the transition may long months :;). See https://github.com/telefonicaid/fiware-orion/issues/959